src/app/user-management/user.management.component.html

```diff
@@ -33,13 +33,19 @@ <h4 *ngIf="canUpdate" class="modal-title w-100 font-weight-bold">Update User</h4
           </select>
         </div>
 
+
         <div class="md-form mb-4">
           <mdb-icon fas icon="lock" class="prefix grey-text"></mdb-icon>
           <input type="password" id="defaultForm-pass" formControlName="password" class="form-control" mdbInput
             mdbValidate>
           <label for="defaultForm-pass">Password</label>
         </div>
 
+        <div class="md-form mb-4">
```

Comment:
checkbox isssue

Reply:
lets resolve this issue and chips issue in our next deployment.